### PR TITLE
Adding Kafka Sink

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

starting work to get the kafka sink in place

Next: openshift release repo: image build

### CODE REVIEW

**Comment by matzew** [comment]
Failure is expected - since we do not build all we want yet

**Review by pierDipi — APPROVED** [review]
/lgtm

File: openshift/release/knative-eventing-kafka-broker-dp-ci.yaml [hunk 1]
```diff
@@ -403,3 +403,244 @@ spec:
       protocol: TCP
       targetPort: 9090
 ---
+---
```
Comment (pierDipi) [inline openshift/release/knative-eventing-kafka-broker-dp-ci.yaml:406]:
```suggestion
```

File: openshift/release/knative-eventing-kafka-broker-cp-ci.yaml [hunk 2]
```diff
@@ -12,6 +12,167 @@ data:
...
+        - name: Age
+          type: date
+          jsonPath: .metadata.creationTimestamp
+        - name: Ready
+          type: string
+          jsonPath: ".status.conditions[?(@.type==\"Ready\")].status"
+        - name: Reason
+          type: string
+          jsonPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+---
```
Comment (pierDipi) [inline openshift/release/knative-eventing-kafka-broker-cp-ci.yaml:175]:
```suggestion
```

File: openshift/release/knative-eventing-kafka-broker-dp-ci.yaml (hunk 1 above)
Comment (matzew) [reply openshift/release/knative-eventing-kafka-broker-dp-ci.yaml:406]: these are autogenerated 
the generator is not yet fully in place

so I ignore them for now

**Pull request merged**